templates/advantage/index.html

```diff
@@ -19,8 +19,13 @@
     </script>
 
     <div class="row">
-      <div class="col-12">
-        <h1>Your subscriptions</h1>
+      <div class="col-12 p-subscriptions__header">
+        <h1 class="u-no-margin--bottom">Your subscriptions</h1>
+        <div class="p-subscriptions__header-buttons">
+          <a class="p-button--positive" href="/advantage/subscribe{% if get_test_backend %}?test_backend=true{% endif %}">Buy new subscription</a>
+          <a class="p-button--neutral" href="/account/invoices{% if get_test_backend %}?test_backend=true{% endif %}">Invoices</a>
+          <a class="p-button--neutral" href="/account/payment-methods{% if get_test_backend %}?test_backend=true{% endif %}">Payment methods</a>
+        </div>
```

Comment:
Any reason why the header isn't part of the React app?
Not sure if there would be any drawbacks (except having the header load faster) but you'd be able to use your hook for urls

Reply:
I thought as the header is static I'd leave it in the template so that (like you say) it'll be visible while the content component loads.

If you think it'd be better to have it in React then I'll follow up with another PR to move it over (just so this PR doesn't get too large).